README.md

```diff
@@ -110,64 +106,86 @@ jobs:
     steps:
-      uses: cloudflare/wrangler-action@1.3.0
+      uses: cloudflare/wrangler-action@2.0.0
       with:
         apiToken: ${{ secrets.CF_API_TOKEN }}
+        command: publish
         preCommands: echo "*** pre command ***"
         postCommands: |
           echo "*** post commands ***"
           wrangler kv:key put --binding=MY_KV key2 value2
           echo "******"
 ```
 
-Set the optional `publish` input to false to skip publishing your Worker project and secrets.  Useful in conjunction with pre and post commands.  For example, if you only wanted to run `wrangler build` against your project:
+You can use the `command` option to do specific actions such as running `wrangler whoami` against your project:
 
 ```yaml
 jobs:
   deploy:
     steps:
-      uses: cloudflare/wrangler-action@1.3.0
+      uses: cloudflare/wrangler-action@2.0.0
       with:
         apiToken: ${{ secrets.CF_API_TOKEN }}
-        publish: false
-        preCommands: wrangler build
+        command: whoami
 ```
 
 ## Use cases
 
-### Deploying when commits are merged to master
+### Deploying when commits are merged to main
 
-The above workflow examples have already shown how to run `wrangler-action` when new commits are merged to the master branch. For most developers, this workflow will easily replace manual deploys and be a great first integration step with `wrangler-action`:
+The above workflow examples have already shown how to run `wrangler-action` when new commits are merged to the main branch. For most developers, this workflow will easily replace manual deploys and be a great first integration step with `wrangler-action`:
 
 ```yaml
 on:
   push:
     branches:
-      - master
+      - main
 
 jobs:
   deploy:
     runs-on: ubuntu-latest
     name: Deploy
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v3
       - name: Publish
-        uses: cloudflare/wrangler-action@1.3.0
+        uses: cloudflare/wrangler-action@2.0.0
         with:
           apiToken: ${{ secrets.CF_API_TOKEN }}
+          command: publish
 ```
 
-Note that there are a number of possible events, like `push`, that can be used to trigger a workflow. For more details on the events available, check out the [GitHub Actions documentation](https://help.github.com/en/articles/workflow-syntax-for-github-actions#on).
+Note that there are a number of possible events, like `push`, that can be used to trigger a workflow. For more details on the events available, refer to the [GitHub Actions documentation](https://help.github.com/en/articles/workflow-syntax-for-github-actions#on).
+
+### Deploying Pages site (production & preview)
+
+If you want to deploy your Pages project with GitHub Actions rather than the built-in CI, then this is a great way to do it. Wrangler v2 will populate the commit message and branch for you. You only need to pass the project name. If a push to a non-production branch is done, it will publish as a preview deployment:
```

Comment:
```suggestion
If you want to deploy your Pages project with GitHub Actions rather than the built-in CI, then this is a great way to do it. Wrangler 2 will populate the commit message and branch for you. You only need to pass the project name. If a push to a non-production branch is done, it will publish as a preview deployment:
```

What is a CI? Need to add full noun then (CI) in this first sentence.